docs/semgrep-supply-chain/policies.md

```diff
@@ -0,0 +1,74 @@
+---
+slug: policies
+title: Manage policies
+hide_title: true
+description: Use policies to define the conditions in which developers are notified of a finding or potentially blocked from merging their PR or MR.
+tags:
+  - Semgrep Supply Chain
+  - Semgrep AppSec Platform
+---
+
+# Manage policies (beta)
+
+Use Supply Chain policies to define the conditions in which developers are **notified** of a finding through a comment, or potentially **blocked** from merging their pull or merge request (PR or MR). This feature helps you manage noise and ensures that developers are only notified or potentially blocked based on the conditions you set.
+
+This feature enables you to configure the following:
+
+- **Scope**: These are the projects (repositories) that are affected by the policy.
+- **Conditions**: The conditions under which **actions** are performed. These conditions are typically related to the value of a finding's severity or reachability.
```

Comment:
I liked that you included examples of what could be an example of a condition, though I wonder if we could frame it as something like examples of finding attributes that can be defined as a condition. So something like the following maybe?
```suggestion
- **Conditions**: The conditions under which **actions** are performed. These conditions are typically attributes of a finding such as severity or reachability. 
```